api/Dockerfile

```diff
@@ -4,8 +4,8 @@
 ## SPDX-License-Identifier: MIT
 ##
 
-ARG BUILD_BASE_IMAGE=golang:1.22.3-alpine
-ARG TARGET_BASE_IMAGE=ubuntu:latest
+ARG BUILD_BASE_IMAGE=golang:1.22.4-alpine
+ARG TARGET_BASE_IMAGE=ubuntu:24.04
```

Comment:
Is there a specific reason to use this version?

Reply:
There are no reason. (personal preference)
This was a personal test environment setting.
I created a commit for deletion and the PR will be updated.